docs/content/faq/developers.mdx

```diff
@@ -25,7 +25,7 @@ Once you have connected to an access node, you can fetch information regarding a
 
 #### Accounts, Contracts, Blocks, Collections, Transactions, Events
 
-These types are documented on the [Access API](https://docs.onflow.org/access-api) page.
+These types are documented on the [Access API](https://docs.onflow.org/http-api) page.
```

Comment:
We are migrating to flow.com domain so it's better to specify links with the new domain to avoid redirects
https://developers.flow.com/http-api